src/core/toga/widgets/splitcontainer.py

```diff
@@ -55,17 +63,26 @@ def content(self, content):
         self._content = []
         for position, item in enumerate(content):
             if isinstance(item, tuple):
-                widget, weight = item
+                if len(item) == 5:
```

Comment:
This will break any existing code using weighted splits. I would suggest that we should accomodate tuples of length 2 *and* 3 (3, rather than 5, because the shrink and wide_handle options aren't required). Tuples of length 2 would default to `flex=True`.